lib/models/ObjectMD.js

```diff
@@ -352,6 +359,50 @@ class ObjectMD {
         return this._data['content-md5'];
     }
 
+    /**
+     * Set content-language
+     *
+     * @param {string} contentLanguage - content-language
+     * @return {ObjectMD} itself
+     */
+    setContentLanguage(contentLanguage) {
+        this._data['content-language'] = contentLanguage;
+        return this;
+    }
+
+    /**
+     * Returns content-language
+     *
+     * @return {string} content-language
+     */
+    getContentLanguage() {
+        return this._data['content-language'];
+    }
+
+    /**
+     * Set Creation Date
+     *
+     * @param {string} creationTime - Creation Date
+     * @return {ObjectMD} itself
+     */
+    setCreationTime(creationTime) {
+        this._data['creation-time'] = creationTime;
+        return this;
+    }
+
+    /**
+     * Returns Creation Date
+     *
+     * @return {string} Creation Date
+     */
+    getCreationTime() {
+        // If creation-time is not set fallback to LastModified
+        if (!this._data['creation-time']) {
```

Comment:
From my understanding of the constructor code, this condition is never met, or if it is, last-modified is not defined either. Am I wrong?